src/main/java/frc/robot/RobotContainer.java

```diff
@@ -37,9 +40,9 @@ public RobotContainer() {
 
   private void configureBindings() {
     conDriver.btn_B.onTrue(Commands.runOnce(() -> subDrivetrain.resetModulesToAbsolute()));
-    conDriver.btn_Start.onTrue(Commands.runOnce(() -> subDrivetrain.resetYaw(180)));
+
     conDriver.btn_Back.onTrue(
-        Commands.runOnce(() -> subDrivetrain.resetPoseToPose(new Pose2d(1.35, 5.50, Rotation2d.fromDegrees(180)))));
+        Commands.runOnce(() -> subDrivetrain.resetPoseToPose(constField.getFieldPositions().get()[6].toPose2d())));
```

Comment:
I think we need to bump the SuperCORE version to use this, this way

Reply:
yes yes yes this does need to happen